src/css/page_sections/Menu/_MenuItem.scss

```diff
@@ -1,91 +1,88 @@
 .coa-MenuItem {
-  border-bottom: 1px solid $color-gray;
-  text-transform: uppercase;
+  cursor: pointer;
+  border-bottom: 1px solid $coa-color-gray;
   font-size: $coa-font-size-small;
   font-family: $coa-font-sans;
   line-height: $coa-line-height-small;
-  position: relative;
-  margin: 0;
-  cursor: pointer;
   padding: $coa-space-level2 $coa-space-level3;
+  text-transform: uppercase;
   @include media($coa-large-screen) {
     border: 1px solid transparent;
-    padding: $coa-space-level4 $coa-space-level2 $coa-space-level3;
-  }
-  > a {
-    color: $coa-color-blue;
-  }
-  svg {
-    transition-duration: .3s;
-  }
-  &:hover, &:focus {
-    .coa-MenuItem__text {
-      @include media($coa-large-screen) {
-        border-bottom: 3px solid $coa-color-orange;
-      }
-    }
+    border-bottom: none;
   }
 }
 
 .coa-MenuItem--open {
+  border-bottom: 1px solid transparent;
   @include media($coa-large-screen) {
-    border-color: $color-gray-medium;
-    border-bottom: 1px solid transparent;
-  }
-  .coa-MenuItem__text {
-    color: $coa-color-orange;
-    border-bottom: 3px solid transparent !important;
-  }
-  .coa-Submenu {
-    display: block;
-    @include media($coa-large-screen) {
-      position: absolute;
-    }
-  }
-  .coa-MenuItem__arrow-down {
-    svg {
-      transform: rotate(-180deg);
-    }
-    path {
-      fill: $coa-color-orange;
-    }
-  }
-}
-
-.coa-MenuItem--flex {
-  display: flex;
-  a {
-    flex-shrink: 0;
-    text-decoration: underline;
-    color: $coa-color-blue;
+    background: $coa-color-white;
+    border: 1px solid $color-gray-medium;
+    border-bottom: none;
+    position: relative;
+    z-index: 1;
   }
 }
 
 .coa-MenuItem--small {
   font-size: $coa-font-size-xsmall;
-}
+  white-space: nowrap;
 
-.coa-MenuItem--home > a {
-  color: $coa-color-orange;
-  text-decoration: none;
+  &.coa-MenuItem--home > a {
```

Comment:
I don't think this should be nested unless is needs to be. It just adds extra specificity and makes it harder to overwrite the rule if needed

Reply:
🌮